scene/resources/animation.cpp

```diff
@@ -5239,9 +5239,7 @@ bool Animation::_fetch_compressed(uint32_t p_compressed_track, double p_time, Ve
 
 	double page_base_time = compression.pages[page_index].time_offset;
 	const uint8_t *page_data = compression.pages[page_index].data.ptr();
-#ifndef _MSC_VER
-#warning Little endian assumed. No major big endian hardware exists any longer, but in case it does it will need to be supported
-#endif
+	// Little endian assumed. No major big endian hardware exists any longer, but in case it does it will need to be supported.
```

Comment:
I'm opening issues to keep track of most `#warning`s converted to TODOs, but for this one I think we don't need it. This is just a reminder that if/when big endian support is needed for animations, this is where it should be implemented.

@godotengine/android